src/main/kotlin/blackjack/dealer/Dealer.kt

```diff
@@ -0,0 +1,16 @@
+package blackjack.dealer
+
+import blackjack.player.Player
+
+class Dealer {
+
+    fun dealInitialCards(players: List<Player>) {
+        players.forEach { player ->
+            drawCard(player)
+            drawCard(player)
+        }
+    }
```

Comment:
이렇게 작성하면 `drawCard` 를 2번 나누어준다는 것을 더 명시적으로 나타내면서 중복 코드를 줄일 수 있을 것 같아요 :) 
```suggestion
    fun dealInitialCards(players: List<Player>) {
        players.forEach {
            dealInitialCards(it)
        }
    }

    private fun dealInitialCard(player: Player) {
        repeat(INITIAL_CARD_COUNT) { drawCard(player) }
    }
```